src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.Scanner;
 public class Duke {
+    public static final String MARKDONE = "mark ";
+    public static final String MARKUNDONE = "unmark ";
```

Comment:
Great use of static variables. Just one issue, shouldn't the naming be MARK_DONE and MARK_UNDONE instead?